pkg/shares/sparse_shares_test.go

```diff
@@ -126,47 +126,86 @@ func TestParsePaddedMsg(t *testing.T) {
 		},
 	}
 	msgs.SortMessages()
-	msgWr.Write(msgs.MessagesList[0])
-	msgWr.WriteNamespacedPaddedShares(4)
-	msgWr.Write(msgs.MessagesList[1])
-	msgWr.WriteNamespacedPaddedShares(10)
-	shares := msgWr.Export()
+	sss.Write(msgs.MessagesList[0])
+	sss.WriteNamespacedPaddedShares(4)
+	sss.Write(msgs.MessagesList[1])
+	sss.WriteNamespacedPaddedShares(10)
+	shares := sss.Export()
 	rawShares := ToBytes(shares)
 	pmsgs, err := parseSparseShares(rawShares, appconsts.SupportedShareVersions)
 	require.NoError(t, err)
 	require.Equal(t, msgs.MessagesList, pmsgs)
 }
 
-func TestMsgShareContainsInfoByte(t *testing.T) {
-	sss := NewSparseShareSplitter()
-	smallMsg := generateRandomMessage(appconsts.SparseShareContentSize / 2)
-	sss.Write(smallMsg)
-
-	shares := sss.Export()
-
-	got := shares[0][appconsts.NamespaceSize : appconsts.NamespaceSize+appconsts.ShareInfoBytes][0]
-
-	isSequenceStart := true
-	want, err := NewInfoByte(appconsts.ShareVersion, isSequenceStart)
```

Comment:
Apologies for including a refactor to the info byte tests while implementing `TestSparseShareSplitterCount`. The refactor can be extracted to a separate PR if requested.